### PR TITLE
pkg addition: Wiko packages

### PR DESCRIPTION
**Pull request by jxd1337**

### CODE REVIEW

**Review by Frigyes06 — APPROVED** [review]

**Comment by Frigyes06** [comment]
Do you want to check Wiko as supported in readme?

**Comment by jxd1337** [comment]
Sure, even though it's definitely not a lot of packages, but better than nothing.

**Comment by Frigyes06** [comment]
Alright. I'll merge after you update the readme :+1:

**Comment by jxd1337** [comment]
Updated ^

**Comment by Frigyes06** [comment]
Hmmm. It appears your packages are missing the dependencies field for each entry.

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -701,6 +701,150 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.ape.walkfit",
+    "list": "Oem",
+    "description": "Walk counter and health for Wiko mobile",
+    "neededBy": [],
```
Comment (Frigyes06) [inline resources/assets/uad_lists.json:708]: "dependencies": [],
on line above neededBy

**Comment by jxd1337** [comment]
Totally mised that! Fixed.

**Comment by Frigyes06** [comment]
Thanks!

**Pull request merged**